src/Math/Math.jl

```diff
@@ -154,4 +154,7 @@ end
     )
 end
 
+# TODO (cg 2023/05/16 18:09): dispute this cutoff.
```

Comment:
Based on some quick checks...
```julia
julia> [1 - besselk_power_series(v, x) /  besselk(ArbFloat(v), ArbFloat(x)) for v in [1e-12, 1e-10, 1e-8, 1e-7, 1e-6, 1e-5, 1e-4, 1e-3, 1e-2, 1e-1, 0.5]]
11-element Vector{ArbFloat{152}}:
 -2.437748225706634247990838067198471067e-5
 -1.5248356887398531378908124154655142622e-6
 -8.8030481136177616693401505579293356406e-9
 -3.0215319688686805981962399367100942475e-10
 -4.2307046642362024167022119670636168767e-11
  1.8747860408766777130173662353255600882e-12
  1.4105941783236156741353136777231924369e-13
  2.4603845442581853417449912598606738772e-14
  2.9431567407118118671190142959485468972e-16
  1.112169174162631706968883320053396081e-16
  3.9750538820132704695818503097085830205e-17

julia> [1 - besselk_powerseries_temme(v, x) /  besselk(ArbFloat(v), ArbFloat(x)) for v in [1e-12, 1e-10, 1e-8, 1e-7, 1e-6, 1e-5, 1e-4, 1e-3, 1e-2, 1e-1, 0.5]]
11-element Vector{ArbFloat{152}}:
  4.0780888276489240281643096774602531073e-17
  4.0791977343079937401306325973013436729e-17
 -1.0169115951304676213730857661491958559e-16
 -9.3923218661627316703162900868338517465e-15
 -9.3604834157217869231377693622750929203e-13
  5.7857370987294525370310780647890426155e-12
  1.9110073981385827390213859944947724355e-13
 -4.8747783965131792183319615081628517939e-14
  2.9431567407118118671190142959485468972e-16
  3.6179964838799983866890363251240805112e-16
 -5.0000000000000001485887462374656391042
```
Anything say `nu < 1e-5` the regular power series suffers from bad cancellation. This method does ok there but then improves for closer values to 0.0.

For values close to 1.0
```julia
julia> [1 - besselk_power_series(v+1, x) /  besselk(ArbFloat(v+1), ArbFloat(x)) for v in [1e-12, 1e-10, 1e-8, 1e-7, 1e-6, 1e-5, 1e-4, 1e-3, 1e-2, 1e-1, 0.5]]
11-element Vector{ArbFloat{152}}:
  3.7036484071836443872668767822899565384e-8
  3.699018487908239799083626088163072763e-8
  3.0688571004728984343457079340533804273e-10
  3.5594453376945422639084151644580242357e-11
  8.1692440278528648751050432825646433798e-13
 -1.108129603597269150133248738240719468e-13
  4.5034265792737568727643784043023925945e-14
  3.5540476887460147608629304537103873234e-15
 -6.5985867193744656152405846396638353378e-17
 -2.5237881226105830131929379845367533749e-16
  5.8006575987128165309263625188029627963e-17

julia> [1 - besselk_powerseries_temme(v+1, x) /  besselk(ArbFloat(v+1), ArbFloat(x)) for v in [1e-12, 1e-10, 1e-8, 1e-7, 1e-6, 1e-5, 1e-4, 1e-3, 1e-2, 1e-1, 0.5]]
11-element Vector{ArbFloat{152}}:
  2.4678131728588103761840070058261918692e-17
  1.944461617643002900191084218027091648e-16
  2.6861655577956104010865642448634667857e-16
  3.5633789608433126654729895039029486644e-16
  3.437217947251377348953394961769863316e-14
  9.4863956442893081566223411847360460185e-14
  1.2123956071699460311658479987681255782e-14
  5.8401517272697592275518011561218324929e-16
  2.9916437547028997470263679786871563016e-16
 -2.5237881226105830131929379845367533749e-16
 -7.102408373229246308596384950751581599e-17
```

So I think any value `modf(x)[1] < 1e-5` we should use this method!